flake.nix

```diff
@@ -331,8 +330,8 @@
             libp2p_helper kimchi_bindings_stubs snarky_js validation trace-tool
             zkapp-cli;
           inherit (dockerImages)
-            mina-image-slim mina-image-full mina-archive-image-full
-            mina-image-instr-full;
+            zeko-image-full mina-image-slim mina-image-full
```

Comment:
Is this just formatting?

Reply:
Added zeko-image-full here, the rest - formatter changes